test/controllers/u2f_registrations_controller_test.rb

```diff
@@ -56,17 +56,38 @@ def canned_u2f_response
     assert_redirected_to two_factor_registrations_path, "redirects to two_factor_registrations"
   end
 
-  test "index renders registered keys" do
-    publisher = publishers(:verified)
+  test "index renders a registered key" do
+    publisher = publishers(:completed)
+    u2f_registration = u2f_registrations(:default)
+    publisher.u2f_registrations << u2f_registration
+
+    sign_in publisher
+    get two_factor_registrations_path
+
+    assert_response :success
+    assert_match u2f_registration.name, response.body
+    assert_match /Set up an authenticator as\sthe secondary 2FA/, response.body
```

Comment:
minor: it may be better to check for tags with [data-test-id] in case strings change